auraxium/event/_trigger.py

```diff
@@ -213,7 +213,7 @@ def generate_subscription(self, logical_and: Optional[bool] = None) -> str:
         # set the "logicalAnd*" flag to avoid subscribing to all characters on
         # all continents (characters would default to "all" if not specified).
         elif (self.worlds and not self.characters
-                and any((issubclass(e, CharacterEvent))  # type: ignore
+                and any((isinstance(e, CharacterEvent))  # type: ignore
```

Comment:
This works, but it would also break auto-insertion of the logicalAnd field as `e` is either a string or type.

The suggestion below does retain logicalAnd auto-insertion for events passed as types, but will reject any string event type.
```suggestion
              and any(not isinstance(e, str) and issubclass(e, CharacterEvent)
                      for e in self.events)):
```

Reply:
Fixing this properly requires some larger refactoring; such as moving `event_factory()` out of `event/_client.py` and using it here to convert the string representation into its class variant (including custom `GainExperience_*` variants) for the proper issubclass check.

Either way, those changes would be beyond the scope of this issue, so I think the above will do for now.